backend-tests/run

```diff
@@ -36,6 +43,8 @@ PYTEST_REPORT_ENTERPRISE="--self-contained-html \
         --html=report_backend_integration_enterprise.html"
 PYTEST_REPORT=""
 
+PYTEST_EXTRA_ARGS=""
+
```

Comment:
Unused. This functionality is already captured with `USER_PYTEST_ARGS`

Reply:
It is used only when the `AZURE_STORAGE_CONTAINER_PREFIX` env. variable is set. I can reuse the existing `USER_PYTEST_ARGS` if that's preferred.

Reply:
Oh, I didn't realize. Yeah, I think it is cleaner if we reuse what we have :+1:

Reply:
I did some [small refactoring](https://github.com/mendersoftware/integration/compare/9979b7cbb6cef8732f564c42ae3a4c2ef6375cd7..d2941d4a1dfd3b71d7b3e40b390496fd9f0064fa) of the pytest arguments for the run script removing the `PYTEST_EXTRA_ARGS` variable.